src/Microsoft.Crank.Agent/Startup.cs

```diff
@@ -4153,6 +4153,36 @@ private static async Task<(string version, string hash)> ParseLatestVersionFile(
             }
         }
 
+        /// <summary>
+        /// Parses files that contains lines with [prefix]:[sha], [version]
+        /// </summary>
+        private static async Task<(string version, string hash)> ParseVersionsFile(string urlOrFilename, string prefix)
+        {
+            var content = urlOrFilename.StartsWith("http", StringComparison.OrdinalIgnoreCase)
```

Comment:
Instead of the StartsWith, can we use something like [Uri.IsWellFormedUriString](https://docs.microsoft.com/en-us/dotnet/api/system.uri.iswellformeduristring?redirectedfrom=MSDN&view=net-6.0#System_Uri_IsWellFormedUriString_System_String_System_UriKind_)?